pipeline/src/hooks.server.js

```diff
@@ -1,6 +1,6 @@
 import { redirect } from '@sveltejs/kit';
 
-const PROTECTED_ROUTES = ['/explore', '/profile', '/createProject']; // Add your protected routes here
+const PROTECTED_ROUTES = ['/explore', '/profile', '/createProject', '']; // Add your protected routes here
 const PUBLIC_ROUTES = ['/signUp', '/signIn'];
```

Comment:
I wonder if we can have one of these implicit.

e.g. we just maintain protected routes. and any route that isn't protected is automatically treated as public